docker/production.Dockerfile

```diff
@@ -75,6 +84,9 @@ COPY --from=builder /app/server/dist ./app/server/dist
 RUN chgrp -R 0 /app && chmod -R g=u /app
 WORKDIR /app
 # Dependencies for scripts outside nestjs
+RUN apt-get install -y nodejs
```

Comment:
nodejs is already installed above right?
https://github.com/ToolJet/ToolJet/pull/5396/files#diff-b62d953b88aedfb7f9adc57bdef9195d7a0280a2ed075f495bd58622cad73eadR42

And if I recall correctly npm also comes with it.